routes/users.route.ts

```diff
@@ -0,0 +1,85 @@
+import { Request, Response, Express } from 'express';
+
+import { getUser, transformUsersToResponceObj } from '../servises/users.service';
+import { UserResponse } from '../models/users';
+import { getEvent, transformEventToResponceObj } from '../servises/events.service';
+
+module.exports = (app: Express): void => {
+  app.get('/get-artists-by-query', getArtistWithNextShow);
+  app.get('/get-artists-amount', getArtistsAmount);
+};
+
+interface UsersQueryObj {
+  active?: boolean;
+  _id?: string;
```

Comment:
Are "active, username, _id" should be "?"?

Reply:
yes, they are. I use this parameters for dynamically filter for find users.